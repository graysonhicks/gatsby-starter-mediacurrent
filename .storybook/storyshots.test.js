import initStoryshots from '@storybook/addon-storyshots'

// Mock functions here.
// jest.mock('../src/global/hooks/useMyHook', () => {
//   return {
//     useMyHook: () => {}
//   };
// });

initStoryshots()
