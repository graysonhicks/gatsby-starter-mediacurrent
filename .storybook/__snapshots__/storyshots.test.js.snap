// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Foo Default 1`] = `
<div
  className="foo"
  data-testid="foo"
>
  The site url is 
  https://dev.mysite.com
</div>
`;

exports[`Storyshots Global/Colors Default 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <h3>
    Brand Colors
  </h3>
  <ul
    style={
      Object {
        "display": "grid",
        "gridGap": "20px",
        "gridTemplateColumns": "repeat(auto-fill, minmax(120px, 175px))",
        "marginBottom": "40px",
      }
    }
  />
  <h3>
    Accent Colors
  </h3>
  <ul
    style={
      Object {
        "display": "grid",
        "gridGap": "20px",
        "gridTemplateColumns": "repeat(auto-fill, minmax(120px, 175px))",
        "marginBottom": "40px",
      }
    }
  />
  <h3>
    Neutral Colors
  </h3>
  <ul
    style={
      Object {
        "display": "grid",
        "gridGap": "20px",
        "gridTemplateColumns": "repeat(auto-fill, minmax(120px, 175px))",
        "marginBottom": "40px",
      }
    }
  />
</div>
`;

exports[`Storyshots Global/Spacing Default 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
/>
`;

exports[`Storyshots Global/Typography/Headings Headings 1`] = `
Array [
  <h1>
    Heading 1
  </h1>,
  <h2>
    Heading 1
  </h2>,
  <h3>
    Heading 1
  </h3>,
  <h4>
    Heading 1
  </h4>,
  <h5>
    Heading 1
  </h5>,
  <h6>
    Heading 1
  </h6>,
]
`;

exports[`Storyshots Global/Typography/Paragraph Paragraph 1`] = `
<p>
  Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat dolorum laborum minus facilis deserunt repudiandae debitis, officia commodi omnis! Animi commodi excepturi blanditiis dicta facere minus quasi fugit maxime atque?
</p>
`;
